### PR TITLE
b2c customer forgotton password

### PR DESCRIPTION
**Pull request by himanshu-saxena-metacube**
## PR Description

CC-16261 b2c customer forgotton password endpoints

### CODE REVIEW

File: tests/api/b2c/glue/customer_forgotten_password/negative.robot [hunk 1]
```diff
@@ -0,0 +1,30 @@
+*** Settings ***
+Suite Setup    SuiteSetup
```
Comment (OKrivtsova) [inline tests/api/b2c/glue/customer_forgotten_password/negative.robot:2]: please fix folder structure, it should be tests/api/b2c/glue/return_endpoints/return-reasons - and there should be your positive and negative files

File: tests/api/b2c/glue/customer_forgotten_password/negative.robot [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+*** Test Cases ***
+Forgot_password_wrong_email_format
+    I send a POST request:    /customer-forgotten-password    {"data":{"type":"customer-forgotten-password","attributes":{"email":"xyz"}}}
+    Response status code should be:    422
+    And Response reason should be:    Unprocessable Entity
+    And Response should return error code:    901
+    And Response should return error message:    email => This value is not a valid email address.
+    And Response header parameter should be:    Content-Type    ${default_header_content_type}
+
+# Forgot_password_empty_email
```
Comment (OKrivtsova) [inline tests/api/b2c/glue/customer_forgotten_password/negative.robot:17]: why is test name commented?

File: tests/api/b2c/glue/customer_forgotten_password/negative.robot (hunk 1 above)
Comment (himanshu-saxena-metacube) [reply tests/api/b2c/glue/customer_forgotten_password/negative.robot]: fixed

File: tests/api/b2c/glue/customer_forgotten_password/negative.robot (hunk 2 above)
Comment (himanshu-saxena-metacube) [reply tests/api/b2c/glue/customer_forgotten_password/negative.robot]: fixed

**Review by OKrivtsova — APPROVED** [review]

**Pull request merged**